input/delimiter/fallback_delimiter_test.go

```diff
@@ -1,14 +1,14 @@
-package input
+package delimiter
 
 import (
 	"testing"
 )
 
 /*
- * Delimiter tests.
+ * FallbackDelimiter tests.
```

Comment:
The only thing I did here, was renaming the Delimiter (now FallbackDelimiter) and changing the package name.